lib/site_settings/deprecated_settings.rb

```diff
@@ -45,7 +45,7 @@ module SiteSettings::DeprecatedSettings
       false,
       "3.3",
     ],
-    ["min_first_post_typing_time", "fast_typing_threshold", false, "3.4"],
```

Comment:
@lis2: Did you mean to remove this line from the deprecated settings list?

Reply:
Great catch, thank you 💯 

Fixed here 

https://github.com/discourse/discourse/pull/31870